website/addons/dropbox/static/tests/dropboxNodeConfig.test.js

```diff
@@ -0,0 +1,54 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+var assert = require('chai').assert;
+
+var utils = require('tests/utils');
+var faker = require('faker');
+var dropboxNodeConfig = require('../dropboxNodeConfig');
+
+describe('dropboxNodeConfig', () => {
+    var endpoints = [
+        {
+            method: 'GET',
+            url: '/api/v1/12345/dropbox/config/',
+            response: {
+                result: {
+                    ownerName: faker.name.findName(),
+                    userisOwner: true,
+                    userHasAuth: true,
+                    validCredentials: true,
+                    nodeHasAuth: true,
+                    urls: {
+                        owner: '/abc123/',
+                        config: '/api/v1/12345/dropbox/config/'
+                    }
+                }
+            }
+        }
+    ];
+
+    var server;
+    before(() => {
+        server = utils.createServer(sinon, endpoints);
+    });
+
+    after(() => {
+        server.restore();
+    });
+
+    describe('ViewModel', () => {
+        it('fetches data from the server on initialization', (done) => {
+            var vm = new dropboxNodeConfig._ViewModel('/api/v1/12345/dropbox/config/', '', '', function() {
```

Comment:
Also, we need to call `server.respond()` after constructing the `ViewModel`, so it wouldn't be feasible to `return` before that.

However, I just found that sinon's `fakeServer` has an `autoRespond` option, which will call `server.respond()` after request. I'll try that